frontend/src/concepts/areas/useFetchDsciStatus.ts

```diff
@@ -0,0 +1,24 @@
+import axios from 'axios';
```

Comment:
Why `axios` and not simply `fetch`?

Reply:
I'm not even sure we have something tracked for this, but all the calls made to the backend are performed with axios, which we should simply replace with fetch I agree there, but I would rather keep this with axios due to consistency and refactor all calls at once

Reply:
There's a mix. So I thought we would be going with `fetch`.
Fair enough to tackle axios as a whole in future.